### ISSUE TITLE
SA828 support. 

### ISSUE THREAD

**Issue by sysadminpower2019** [0x9900/SA818#20]
Hello,

thanks for your work on this project. Do you think this is compatible also with the sa828 or that it could be possibly made compatible?

thanks

**Comment by 0x9900** [comment]
I don’t think that program will work with the SA828. From what I saw in their datasheet, the SA828 has an entirely different instruction set.
G-NiceRF also sells the module SA868, which should work with this program. The dataset is the same; it just has a couple of more instructions. I have ordered a couple of SA868 to test my program. Unfortunately, I don’t have any SA828 at the moment. I might order one and write a different program for it. 

Thanks for your email, and I am glad you find my program helpful. 

73,
-fred-
